main/scripts/edit_panel.py

```diff
@@ -159,6 +161,8 @@ def create_edit_panel_widgets(self):
 
     def update_slider_value(self, event):
         current_option = self.edit_combobox.get()
+        if not hasattr(self, "current_image"):
+            self.current_image = self.parent.original_image.copy()
```

Comment:
I couldn't find anywhere this attribute is properly initialized and couldn't initialize it in the class's init

Reply:
>Did you encounter any errors related to this attribute?

If you toggle the edit panel and try to change the combobox without touching anything else, `EditPanel` will throw an attribute error. Changing the slider or checking the cumulative edit checkbox was the only place I could see current_image being initialized locally.

Testing it again now, the Revert button will also throw an attribute error for `edit_is_reverted_var` in a similar way, not being initialized in `EditPanel` until `_reset_edit` is run. I actually can't see anywhere it gets initialized in `EditPanel` because the only time `_reset_edit` runs is when that button is clicked while in the "restore" state which is unreachable. Though I remember at some point being able to use it, I'm not sure how lol.

In both cases, checking the attribute on parent fixes the issue.

Reply:
Ok. I'm made the changes to access `current_image`, and `edit_is_reverted_var` from "parent" (ImgTxtViewer), this should resolve the issue.